go/cli/mcap/utils/priority_queue.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"container/heap"
+
+	"github.com/foxglove/mcap/go/mcap"
+)
+
+type PriorityQueue []TaggedMessage
+
+func (pq PriorityQueue) Len() int {
+	return len(pq)
+}
+
+func (pq PriorityQueue) Less(i, j int) bool {
+	return pq[i].Message.LogTime < pq[j].Message.LogTime
```

Comment:
this should do a secondary comparison on import ID so ties are broken deterministically